common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -98,267 +98,360 @@
 public class GTBlocks {
 
     //////////////////////////////////////
-    //*****     Material Blocks    *****//
+    //*****     Tables Builders    *****//
     //////////////////////////////////////
+    // TODO: Remove builders after initialization
```

Comment:
Not 100% sure whether they actually need to be fields - might be able to keep them local in the methods

Reply:
These will be depreciated in effect when the initialization process is abstracted like GTItems